src/workspace.ts

```diff
@@ -83,4 +83,36 @@ export class MonacoWorkspace implements Workspace {
         return this.onDidChangeTextDocumentEmitter.event;
     }
 
+    public applyEdit(workspaceEdit: WorkspaceEdit): Thenable<boolean> {
+        let applied = true;
+        if (workspaceEdit.documentChanges) {
```

Comment:
please use [asWorkspaceEdit](https://github.com/TypeFox/monaco-languageclient/blob/master/src/converter.ts#L273), applying `documentChanges` is not enough